romancal/tweakreg/tests/test_tweakreg.py

```diff
@@ -1023,96 +912,6 @@ def test_tweakreg_handles_multiple_groups(tmp_path, base_image):
             res.shelve(r, modify=False)
 
 
-@pytest.mark.parametrize(
-    "column_names",
-    [("x", "y"), ("xcentroid", "ycentroid")],
-)
-def test_imodel2wcsim_valid_column_names(tmp_path, base_image, column_names):
```

Comment:
Are these still useful? I think romancal is still handling the column renaming (although it looks like it could be moved to stcal).

Reply:
Since we moved the validation of the column names to a helper, a corresponding unit test has been implemented to replace the old unit test.